### DISCUSSION TITLE
Add LTS Instructions, and Recommended Install Instruction to Readme

### DISCUSSION THREAD

**Discussion by Katerlad** [FirstGearGames/FishNet#475] [Documentation]
Just a suggestion to Add the LTS methodology explanation and recommended install instructions for first time install as well as ReImporting Fishnet. 

I know there will be an argument that you want as little places to update info as possible, so you may add this in the Docs instead, but keep in mind for new users its intuitive to get Install and version info from the place you are downloading, and then learn how to use the product via docs.

This would be on the ReadMe/ front page of Github as well as probably the Asset Store, These Instructions should probably be in the Text that is shown in the Package manager as well? 

Something maybe like this?

# How to Download Latest
Fishnet uses the Semantic Versioning mixed with identifiers that detail these things detailed below.

X.Y.Z(R)

X <= Major Version, Y <= Minor Version, Z <= Hotfixes, (R) <= Released Branch

Unless you want to test pre-released versions, Make sure you are downloading the most up to date Released branch!
- all versions without a "R" at the end will be an experimental version.

# Recommended Install Instructions

### First Time Install
 - Free Version
 Import the package by adding Fishnet to your assets in the Unity Asset Store, or by downloading the package here in this git repo.
 
 - Pro Version
 If you are a subscriber to FirstGearGames on Patreon or Github, you can download the Pro Versions at www.firstgeargames.com

### Upgrading Versions
- Step1: Backup your project, use source control if you can. 
- Step2: Delete the Fishnet Folder(Assets/Fishnet), You do not have to delete the DefaultPrefabsObject located outside of the Fishnet Folder.
- Warning: After you delete fishnet folder do not restart unity until new package is installed, potentially will lose object references if you do.
- Step3: Import the new fishnet Update using your favorite method. Git, Package Manager, or website for Pro.
- Step4: Restart Unity after the import is complete.

# Release Mode
Info about release mode here.

Does it start in release mode? 
Are you out of release mode as soon as you turn on a experimental feature?

**Comment by Katerlad** [comment]
I feel like ParrelSync is a great example of a Git readme that is clear, and targeted for new users.

https://github.com/VeriorPies/ParrelSync

**Comment by FirstGearGames** [comment]
Much of these steps do not apply anymore. The versioning explanation could be useful, and explaining that Pro/Free can be installed over each other. Putting this under documentation on the road map.
